### DISCUSSION TITLE
Logs from malfunctioning dexcom one collector

### DISCUSSION THREAD

**Discussion by mirantaparthena** [NightscoutFoundation/xDrip#3136] [Bug reports]
Hello, I have uploaded logs from my malfunctioning Dexcom one collector. I have a Cubot king kong mini2 smartphone, Android 10 and my transmitter started to unpair all of sudden. Every 20 minutes it asked for pairing again, got one reading and then unpaired. In th elogs I uploaded it got paired but I didn't get a reading. Transmitter worked fine in dexcom receiver. Can you help? Thank you in advance

**Comment by Navid200** [comment]
The receiver is working fine, right?

**Reply by mirantaparthena** [reply]
Yes receiver got consistent readings

**Comment by jamorham** [comment]
@mirantaparthena are you the same person who previously reported issues like this and with the king kong mini? Has the transmitter been paired with another phone also? I believe this problem can come that the transmitter doesn't like pairing with multiple different phones and fails to properly hold the bluetooth pairing after that. As far as I know it isn't a bug that I can fix and similar problems have been seen with transmitters in other situations as well. (change slot numbers too much etc)

**Comment by mirantaparthena** [comment]
Hello and thank you for replying. Yes, I am the same person. Last time I had a Bluetooth problem as xdrip was trying to connect to my miband and I had disconnection issues, that was addressed by deleting the miband.I had pairing requests this time every 20 minutes and it kept on hunting the transmitter as if it couldn't remember it. I didn't pair it with anything else, at the time it got denied by xdrip (I can see xdrip breaking the bond in the logs) my daughter was trying to Google something with her voice in our tv, that was the only thing happening. Transmitter was working well with the receiver though, while it couldn't connect to xdrip
________________________________
Από: JamOrHam ***@***.***>
Στάλθηκε: Παρασκευή, 20 Οκτωβρίου 2023 4:53 μμ
Προς: NightscoutFoundation/xDrip ***@***.***>
Κοιν.: mirantaparthena ***@***.***>; Mention ***@***.***>
Θέμα: Re: [NightscoutFoundation/xDrip] Logs from malfunctioning dexcom one collector (Discussion #3136)


@mirantaparthena<https://github.com/mirantaparthena> are you the same person who previously reported issues like this and with the king kong mini? Has the transmitter been paired with another phone also? I believe this problem can come that the transmitter doesn't like pairing with multiple different phones and fails to properly hold the bluetooth pairing after that. As far as I know it isn't a bug that I can fix and similar problems have been seen with transmitters in other situations as well. (change slot numbers too much etc)
